content/error-handling/error-handling-assignment/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+content_type: project
+flavours:
+  - java
+  - javascript
+  - markdown
+  - python
+prerequisites:
+  hard:
+    - error-handling/java-error-handling
+    - error-handling/python-error-handling
+    - error-handling/javascript-error-handling
+protect_main_branch: false
+ready: true
+submission_type: repo
+title: Error Handling Assignment
+---
+
+Please answer the following questions in markdown files and submit following the instructions below.
+
+## Questions
+
+1. What is the fundamental purpose of error handling?
+
+2. What are some poor practices in error handling that developers should avoid? Provide 1 example.
```

Comment:
"In your programming language" should be mentioned somewhere in this sentence since we require an example